tests/test_incsyncres.py

```diff
@@ -1,10 +1,11 @@
 from collections import namedtuple
 import logging
+from urllib.parse import urlparse
 import pytest
 import responses
-from urllib.parse import urlparse
 import openqabot.incsyncres
 from openqabot.incsyncres import IncResultsSync
+from openqabot.approver import QEM_DASHBOARD
```

Comment:
I am not sure about this one but I haven't found a way to import variable directly from __init__ :thinking:

Reply:
`from openqabot import QEM_DASHBOARD` ....

Reply:
done